src/jsifier.js

```diff
@@ -410,6 +413,10 @@ function ${name}(${args}) {
         contentText = `var ${mangled} = ${snippet};`;
       }
       const sig = LibraryManager.library[symbol + '__sig'];
+      isAsyncFunction = LibraryManager.library[symbol + '__async'] ?? isAsyncFunction;
```

Comment:
To add new attribute like this you also need to add it to `isJsLibraryConfigIdentifier()` in `src/utility.js`.

If this new attribute useful for normal asyncify?   Can you land the addition of this new attribute as a separate change maybe, along with a simple/existing use case?